generators/php/colour.js

```diff
@@ -39,10 +39,10 @@ Blockly.PHP['colour_random'] = function(block) {
   // Generate a random colour.
   var functionName = Blockly.PHP.provideFunction_(
       'colour_random',
-      [ 'function ' + Blockly.PHP.FUNCTION_NAME_PLACEHOLDER_ + '() {',
-        '  return \'#\' . str_pad(dechex(mt_rand(0, 0xFFFFFF)), ' +
-            '6, \'0\', STR_PAD_LEFT);',
-        '}']);
+      ['function ' + Blockly.PHP.FUNCTION_NAME_PLACEHOLDER_ + '() {',
+       '  return \'#\' . str_pad(dechex(mt_rand(0, 0xFFFFFF)), ' +
```

Comment:
In colour_rgb (20 lines down)  PHP strings are double-quoted, whereas here they are single-quoted.  Let's make this consistent.  Not your fault, it's a preexisting condition in the code.